src/app/screens/gamepage/index.ts

```diff
@@ -178,6 +178,20 @@ export class GamePage {
       })
     })
   }
+  private recruitUnits() {
```

Comment:
do you run this function anywhere??

run this function eg. inside bindEvents function